config/pmd-test.xml

```diff
@@ -191,4 +191,17 @@
         </properties>
     </rule>
 
+    <rule ref="category/java/design.xml/UseObjectForClearerAPI">
+        <properties>
+            <!-- We try to state more specifically on the properties
+                    that users should pass(String files, String checks,
+                    String message, String moduleId, String query) instead
+                    of using containers which might cause confusion while
+                    using SuppressionXpathSingleFilter -->
```

Comment:
commend should be:
`This is checking for amount of arguments more that 3 (hardcoded).  But in make sense to start making violations on items where amount of arguments more that 7. Extra abstraction(object) for single method does not make code more readable or easy to maintain.`

if we get more annoying messages from this rule - https://pmd.github.io/pmd-6.4.0/pmd_rules_java_design.html , we can disable it. But while it is single let keep it.

Reply:
done.